flamby/results/plot_results.py

```diff
@@ -164,9 +189,12 @@ def get_nb_clients_from_dataset(name_arg):
     current_title = title_dicts[name]
     ax.set_title(current_title, fontsize=45, fontweight="heavy")
 
-# Hide the last plot
+# Hide the two plots on the last row
 flattened_axs[-3].set_visible(False)
 flattened_axs[-1].set_visible(False)
+
+# No idea why I need to add that probably because of sharey on the last row
```

Comment:
this comment is about previous or following lines?

Reply:
Good catch !